src/components/EthereumAddress.tsx

```diff
@@ -8,7 +8,7 @@ import { Flex } from './Flex'
 import QRCodeModal from './QRCodeModal'
 
 interface Props {
-  address: string | undefined
+  address: EthAddress | undefined
```

Comment:
This component is a mess. `BLOCKCHAIN_EXPLORER_URL` is no longer available, it is used for transaction IDs, not ethereum addresses, and only used by CashoutModal ?!